MD/md-20/README.md

```diff
@@ -0,0 +1,93 @@
+# MD-20: Bridge Swap Mechanism
+
+**Description**: This MD outlines and proposes the mechanics for the swap to take place, as part of the lock-mint-swap Bridge Transfer.
+
+## Overview
+
+The Bridge uses a lock-mint-swap process to provide our users with native gas-paying L2 $MOVE for their locked L1 ERC20 $MOVE. This transfer occurs in one sequence for the user, within the timelock period. The Lock and Mint phases of the sequence have already been implemented and were proposed in [RFC40](https://github.com/movementlabsxyz/rfcs/tree/main/0040-atomic-bridge). Therefore, these parts of the sequence will not be discussed here, only referenced.
```

Comment:
RFC40 does not define what "sequence" means. Which steps does sequence entail? all 6 that are mentioned in RFC40?